server/profileFormUpdate.js

```diff
@@ -17,11 +18,16 @@ s3.config.update({
 });
 
 //TODO: change the actual PRIVATE_BUCKET name, for whatever reason AWS wasn't working properly and I couldn't change it
+// AWS bucket names
 const LICENSE_BUCKET = "cleaner-licenses";
 const TOOLS_BUCKET = "semi-private";
-
 let s3Urls = ['',''];
 
+//Application Status Names:
+const IN_PROGRESS = 'Began Application';
+const READY_FOR_VEFIFICATION = 'Application Pending Employer Validation';
```

Comment:
Seems a little wordy, I think we can drop the "Application" => "Pending Employer Validation" 
Same thing with "Application Completed Employer Validation" => "Completed Employer Validation"

Reply:
This wordy because this what is set in the enum that you created in the DB. If I change it, this will be a syntax error. If you would like to change the enums, that is covered in ticket 3.

Reply:
I cannot change these unless they are changed in the DB and this is outside the scope of this ticket